prediction_market_agent_tooling/markets/omen/omen_contracts.py

```
@@ -177,8 +185,8 @@ def redeemPositions(
         api_keys: APIKeys,
         collateral_token_address: HexAddress,
         condition_id: HexBytes,
-        parent_collection_id: HexStr,
         index_sets: t.List[int],
+        parent_collection_id: HexStr = build_parent_collection_id(),
```

Ruff

Do not perform function call `build_parent_collection_id` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)